python/cucim/src/cucim/core/operations/color/__init__.py

```diff
@@ -1,6 +1,13 @@
 from .jitter import color_jitter, rand_color_jitter
+from .stain_normalizer import (absorbance_to_image, image_to_absorbance,
+                               normalize_colors_macenko,
+                               stain_extraction_macenko)
 
 __all__ = [
     "color_jitter",
     "rand_color_jitter"
+    "absorbance_to_image",
+    "image_to_absorbance",
+    'stain_extraction_macenko',
```

Comment:
Nit: quotation mark is not consistent. May want to use " instead of ' in line 11 and 12.
Or, please stick to one and apply the same rule to other cases.